stdlib/SparseArrays/test/sparse.jl

```diff
@@ -2290,6 +2290,36 @@ end
     @test adjoint(MC) == copy(adjoint(SC))
 end
 
+@testset "Triangular matrices" begin
+    rng = Random.MersenneTwister(0)
+    n = 1000
+    A = sprand(rng, n, n, 0.01)
+    B = ones(n)
+    MA = Matrix(A)
+    for tr in (identity, adjoint, transpose)
```

Comment:
It would be great if you could put a testset here that prints the loop variables. E.g. just something like
```julia
@testset "tr=$tr, wr=$wr" for
    tr in (identity, adjoint, transpose),
        wr in (UpperTriangular, LowerTriangular, UnitUpperTriangular, UnitLowerTriangular)
```
helps a lot to identify the problem when a test fails.